flask-backend/requirements.txt

```diff
@@ -1,20 +1,31 @@
+alembic==1.6.5
```

Comment:
Remove redundant and duplicate imports

Reply:
@shivanshu1333 sir, these were installed automatically when I installed Flask-Migrate, python-dotenv packages.

Reply:
And some of them were of #173, that I forgot to add previously :)

Reply:
there are multiple duplicate imports like for Flask-Login==0.5.0,
okay you can do one thing, locally delete the requirements.txt file and inside the venv of flask do pip freeze > requirements.txt and push the file

Reply:
sure sir @shivanshu1333, I have deleted requirements.txt and created requirements.txt where venv is located.

Reply:
Done @shivanshu1333 !, I have removed requirements.txt file from flask-backend, now there is only one requirements.txt file in the same level of venv :)